releases.json

```diff
@@ -21,7 +21,7 @@
         {
             "device": "inf2",
             "min_version": "0.0.16",
-            "max_version": "0.0.17",
+            "max_version": "0.0.18",
```

Comment:
Please also include 0.0.18 as a release candidate. I.e. put it into "releases"

Reply:
OK: I did not know what to put there, so I did what was done in the other pull-request for TGI 1.4.0. Hope I did it right.